conf/available_modules.go

```diff
@@ -8,9 +8,10 @@ package main
 import (
 	_ "mig/modules/agentdestroy"
 	_ "mig/modules/file"
+	_ "mig/modules/migoval"
```

Comment:
Renamed migoval (module file name) -> pkg, and oval (module command name) -> pkg